api4/user.go

```diff
@@ -119,6 +119,17 @@ func getUser(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	userTermsOfService, err := c.App.GetUserTermsOfService(user.Id)
```

Comment:
Can you change this to only fetch and return the terms of service if `c.IsSystemAdmin()` or `c.App.Session.UserId == user.Id`? We shouldn't return this information about other users unless requested by a system admin.

Reply:
Done.